v2/bigtable-cdc-to-hbase/src/test/java/com/google/cloud/teleport/v2/templates/BigtableToHbasePipelineIT.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates;
+
+import static com.google.cloud.teleport.v2.templates.utils.TestConstants.colFamily;
+import static com.google.cloud.teleport.v2.templates.utils.TestConstants.colQualifier;
+import static com.google.cloud.teleport.v2.templates.utils.TestConstants.rowKey;
+import static com.google.cloud.teleport.v2.templates.utils.TestConstants.value;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.bigtable.data.v2.models.Range.TimestampRange;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.cloud.teleport.it.TemplateTestBase;
+import com.google.cloud.teleport.it.bigtable.StaticBigtableResourceManager;
+import com.google.cloud.teleport.metadata.TemplateIntegrationTest;
+import com.google.cloud.teleport.v2.templates.BigtableToHbasePipeline.BigtableToHbasePipelineOptions;
+import com.google.cloud.teleport.v2.templates.utils.HbaseUtils;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.hadoop.hbase.HBaseTestingUtility;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.util.Time;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** End to end table that runs the pipeline from Bigtable to Hbase. */
+@Category(TemplateIntegrationTest.class)
+@TemplateIntegrationTest(BigtableToHbasePipeline.class)
+@RunWith(JUnit4.class)
+public class BigtableToHbasePipelineIT extends TemplateTestBase {
```

Comment:
We should test the source filtering in here too

Reply:
Added source filtering test